editor/code_editor.cpp

```diff
@@ -150,7 +150,7 @@ bool FindReplaceBar::_search(uint32_t p_flags, int p_from_line, int p_from_col)
 			text_editor->unfold_line(pos.y);
 			text_editor->set_caret_line(pos.y, false);
 			text_editor->set_caret_column(pos.x + text.length(), false);
-			text_editor->center_viewport_to_caret(0);
+			text_editor->center_viewport_to_caret();
```

Comment:
Slipped a change from a WIP branch on accident here. It doesn't really matter, 0 is the default value anyway.

Reply:
Yeah, it's better not to introduce unrelated changes. Can you please rollback the code editor stuff?